machine/accton/accton_as4510_52t/kernel/platform-as4510_52t.patch

```diff
@@ -0,0 +1,481 @@
+Support for Accton AS4510_52T platform (ARM Cortex-A9 / Hurricane2) networking platform
+Baseline files sourced from Broadcom LDK 3.4.10
+
+diff --git a/arch/arm/boot/dts/accton_as4510_52t.dts b/arch/arm/boot/dts/accton_as4510_52t.dts
+new file mode 100644
+index 0000000..5ef7c60
+--- /dev/null
++++ b/arch/arm/boot/dts/accton_as4510_52t.dts
+@@ -0,0 +1,92 @@
++/*
++ * Accton AS4510_52T Device Tree Source
++ *
++ * Copyright 2015, Accton Technology Corporation.
++ *
++ * This program is free software; you can redistribute  it and/or modify it
++ * under  the terms of  the GNU General  Public License as published by the
++ * Free Software Foundation;  either version 2 of the  License, or (at your
++ * option) any later version.
++ *
++ */
++/dts-v1/;
++/include/ "helix4.dtsi"
++
++/ {
++    model = "accton,as4510_52t";
++    compatible = "accton,as4510_52t";
++
++    aliases {
++        serial0 = &uart0;
++    };
++
++    memory {
++        reg = <0x61000000 0x7f000000>;
```

Comment:
In fact, I had tried to correct the .dts file yesterday.  It hanged at starting kernel so that I turned to reserve 0x0100_0000 like the way in AS4610_54 and AS4610_30.

       Verifying Hash Integrity ... crc32+ OK
       Booting using the fdt blob at 0x704bb094
       Uncompressing Kernel Image ... OK
    boot_prep_linux commandline: quiet console=ttyS0,115200 mem=2032M serial_num=EC1612001765 eth_addr=CC:37:AB:C2:C8:80 boot_re on=rescue
       Loading Device Tree to 63ffb000, end 63fff017 ... OK
    
    Starting kernel ...

I try revising `CONFIG_PHYS_SDRAM_RSVD_SIZE` in AS4610_54 to `0x0` and get the same issue.  At the situation, I specify `mem=2032M` to kernel bootargs that can boot to linux shell.  If the specified size larger than 2032M, issue would still occur.